roles/dtc/common/tasks/ndfc_inventory.yml

```diff
@@ -48,3 +48,8 @@
     inv_list: "{{ inv_config }}"
   register: updated_inv_config
   no_log: true
+
+- name: Credential Retrieval Failed
```

Comment:
Added this because we don't see the error message generated by the action plugin to indicate `username` and `password` is not set because we have the `no_log: true` set.